core/src/test/java/org/apache/carbondata/scan/result/impl/NonFilterQueryScannedResultTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.carbondata.scan.result.impl;
+
+import mockit.Mock;
+import mockit.MockUp;
+import org.apache.carbondata.scan.executor.infos.BlockExecutionInfo;
+import org.apache.carbondata.scan.model.QueryDimension;
+import org.apache.carbondata.scan.result.AbstractScannedResult;
+import org.junit.Before;
+import org.junit.Test;
+
+public class NonFilterQueryScannedResultTest {
+    private static NonFilterQueryScannedResult nonFilterQueryScannedResult;
+
+    @Before
+    public  void setUp(){
+        BlockExecutionInfo blockExecutionInfo = new BlockExecutionInfo();
+        QueryDimension queryDimension[] = {new QueryDimension("dummyColumnName1"),new QueryDimension("dummyColumnName2")};
+        blockExecutionInfo.setQueryDimensions(queryDimension);
+        nonFilterQueryScannedResult = new NonFilterQueryScannedResult(blockExecutionInfo);
+
+    }
+
+    @Test
+    public void testIsNullMeasureValue(){
```

Comment:
this test is doing nothing, please mock it properly. We supposed to set `measureDataChunks` and call this method.